src/models/enums/schedule_type.rs

```diff
@@ -0,0 +1,14 @@
+//! Model for Schedule type enum
+
+use serde::{Deserialize, Serialize};
+
+/// Indicates the fulfillment delivery schedule type. If SCHEDULED, then deliver_at is required.
```

Comment:
I've just been using the documentation from the Square docs for the type:
```rust
/// The schedule type of the delivery fulfillment.
```
See https://developer.squareup.com/reference/square_2022-12-14/enums/OrderFulfillmentDeliveryDetailsScheduleType